Mathlib/Data/Matrix/ColumnRowPartitioned.lean

```diff
@@ -144,6 +144,16 @@ section Semiring
 
 variable [Semiring R]
 
+@[simp]
+lemma fromRows_mulVec (A₁ : Matrix m₁ n R) (A₂ : Matrix m₂ n R) (v : n → R) :
+    fromRows A₁ A₂ *ᵥ v = Sum.elim (A₁ *ᵥ v) (A₂ *ᵥ v) := by
+  ext (_ | _) <;> rfl
+
+@[simp]
+lemma vecMul_fromColumns (B₁ : Matrix m n₁ R) (B₂ : Matrix m n₂ R) (v : m → R) :
+    v ᵥ* Matrix.fromColumns B₁ B₂ = Sum.elim (v ᵥ* B₁) (v ᵥ* B₂) := by
+  ext (_ | _) <;> rfl
```

Comment:
Can you also add the ones where the `Sum.elim` is on the LHS (and the rows/columns are swapped)?

Reply:
I didn't know how to prove the other two.

Reply:
Where do you get stuck? The proof should be trivial still.

Reply:
I committed a sorried lemma. Either provide a proof, or delete it from this PR.

Reply:
I pushed a proof; can you add the other lemma?

Reply:
Thanks!